tests/llm/fixtures/test_workload_health/01_crashpod/kubectl_describe.txt

```diff
@@ -0,0 +1,42 @@
+{"toolset_name":"kubernetes/core","tool_name":"kubectl_describe","match_params":{"kind":"Deployment","name":"crashpod","namespace":"default"}}
+stdout:
+ "Name:                   crashpod
+Namespace:              default
+CreationTimestamp:      Mon, 14 Jul 2025 14:04:46 +0300
+Labels:                 <none>
+Annotations:            deployment.kubernetes.io/revision: 1
+Selector:               app=crashpod
+Replicas:               1 desired | 1 updated | 1 total | 0 available | 1 unavailable
+StrategyType:           RollingUpdate
+MinReadySeconds:        0
+RollingUpdateStrategy:  25% max unavailable, 25% max surge
+Pod Template:
+  Labels:  app=crashpod
+  Containers:
+   crashpod:
+    Image:      busybox
+    Port:       <none>
+    Host Port:  <none>
+    Command:
+      sh
+    Args:
+      -c
+      wget -O - https://gist.githubusercontent.com/odyssomay/1078370/raw/35c5981f8c139bc9dc02186f187ebee61f5b9eb9/gistfile1.txt 2>/dev/null; exit 125;
```

Comment:
Please make sure to remove this from here. We so couple of times that when holmes use describe and he see the commands as part of the describe it effects the actual result.

In real work use cases those args won't be here so better to remove them.